app/Http/Controllers/FenneluckyController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Fennelucky;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class FenneluckyController extends Controller
+{
+    public function store(Request $request): JsonResponse
+    {
+        $request->validate([
+            'name' => 'required|unique:fenneluckies,name,NULL,id,user_id,'.auth()->id(),
+        ]);
+
+        $fennelucky = Fennelucky::create([
+            'name' => $request->name,
+            'user_id' => auth()->id(),
+        ]);
```

Comment:
If you create a new static method inside the model called `::fromRequestInput(array $data)` you can keep all the logic of what the creation needs, what the fillables are, and any other actions that need to be taken (like initializing the pet's stats) inside the model, which is responsible for its logic.

There also exists the completely different approach that says models should **only** contain what's needed for data persistence / retrieval. I do not have a strong preference here at this stage, just offering options.

Reply:
Updated with the suggestion.